src/renderer/components/ft-community-post/ft-community-post.scss

```diff
@@ -21,8 +21,14 @@
 }
 
 .communityImage {
-  block-size: 100%;
-  inline-size: 100%;
+    inline-size: 100%;
+    block-size: 640px;
+    object-fit: cover;
+}
+
+.swiperSlide {
+  max-block-size: 640px;
+  max-inline-size: 640px;
```

Comment:
not sure if a fixed width is desired here (I got no idea how community post UI is designed, pending comments from others

Reply:
every youtube community post displays always at both fixed width and height (at least from my testing)